<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1130</width>
    <height>880</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	font: 700 13pt &quot;Segoe UI&quot;;
	background-color: rgb(13, 183, 100); /*light green*/
	/*border-color: rgb(13, 186, 255); /*light blue*/
	border-radius: 7px;
}

QPushButton:hover {
	background-color: rgb(6, 165, 75); /*darkish green*/
}

QPushButton:pressed {
	background-color:  rgb(15, 121, 63); /*dark green*/
}

QGraphicsView {
	border: 3px solid white;
	border-radius:10px;
}

QListWidget {
	border: 3px solid white;
	border-radius:10px;
}

QLabel {
	color: rgb(255, 255, 255);
	font: 700 13pt &quot;Segoe UI&quot;;
	background-color: rgb(0, 153, 255); /*light blue*/
	border: 3px solid rgb(0, 153, 255); /*light blue*/
	border-radius:10px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QPushButton" name="change_turn_button">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>10</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Change Turn</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QGraphicsView" name="scrabble_board">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>840</width>
      <height>840</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="renderHints">
     <set>QPainter::TextAntialiasing</set>
    </property>
    <property name="dragMode">
     <enum>QGraphicsView::NoDrag</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="undo_button">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>10</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	font: 700 13pt &quot;Segoe UI&quot;;
	background-color: rgb(255, 116, 118); /*light red*/
	border-radius: 7px;
}

QPushButton:hover {
	background-color: rgb(235, 100, 100); /*med red*/
}

QPushButton:pressed {
	background-color:  rgb(200, 50, 50); /*dark red*/
}

QPushButton:disabled {
	background-color:  rgb(255, 180, 180); /*lightest red*/
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/Resources/Menu/undo.png</normaloff>:/Resources/Menu/undo.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="widget_5" native="true">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>60</y>
      <width>261</width>
      <height>161</height>
     </rect>
    </property>
    <widget class="QWidget" name="widget_4" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <widget class="QLabel" name="Score4_text">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-top-right-radius: 0px;
border-top-left-radius: 0px;
border-bottom-right-radius: 0px;</string>
      </property>
      <property name="text">
       <string>Player Four's Score</string>
      </property>
     </widget>
     <widget class="QLabel" name="Score4">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>0</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-top-left-radius: 0px;
border-top-right-radius: 0px;
border-bottom-left-radius: 0px;</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <widget class="QLabel" name="Score1_text">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-top-right-radius: 0px;
border-bottom-right-radius: 0px;
border-bottom-left-radius: 0px;</string>
      </property>
      <property name="text">
       <string>Player One's Score</string>
      </property>
     </widget>
     <widget class="QLabel" name="Score1">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>0</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-top-left-radius: 0px;
border-bottom-left-radius: 0px;
border-bottom-right-radius: 0px;</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <widget class="QLabel" name="Score3_text">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 0px;</string>
      </property>
      <property name="text">
       <string>Player Three's Score</string>
      </property>
     </widget>
     <widget class="QLabel" name="Score3">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>0</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 0px;
</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <widget class="QLabel" name="Score2_text">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 0px;</string>
      </property>
      <property name="text">
       <string>Player Two's Score</string>
      </property>
     </widget>
     <widget class="QLabel" name="Score2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>0</y>
        <width>61</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 0px;</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="undo_button_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>810</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	font: 700 13pt &quot;Segoe UI&quot;;
	background-color: rgb(255, 116, 118); /*light red*/
	border-radius: 7px;
}

QPushButton:hover {
	background-color: rgb(235, 100, 100); /*med red*/
}

QPushButton:pressed {
	background-color:  rgb(200, 50, 50); /*dark red*/
}

QPushButton:disabled {
	background-color:  rgb(255, 180, 180); /*lightest red*/
}</string>
    </property>
    <property name="text">
     <string>  Calibrate</string>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/Resources/Menu/calibrate.png</normaloff>:/Resources/Menu/calibrate.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>23</width>
      <height>22</height>
     </size>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1130</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
